content/influxdb/cloud-iox/visualize-data/grafana.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Use Grafana to visualize data
+seotitle: Use Grafana to visualize data stored in InfluxDB
+list_title: Grafana
+description: >
+  Use [Grafana](https://grafana.com/) to query and visualize data stored in an
+  InfluxDB bucket backed by InfluxDB IOx.
+weight: 101
+menu:
+  influxdb_cloud_iox:
+    name: Grafana
+    parent: Visualize data
+influxdb/cloud-iox/tags: [visualization]
+---
+
+Use [Grafana](https://grafana.com/) to query and visualize data stored in an
+InfluxDB bucket backed by InfluxDB IOx.
+
+> [Grafana] enables you to query, visualize, alert on, and explore your metrics,
+> logs, and traces wherever they are stored.
+> [Grafana] provides you with tools to turn your time-series database (TSDB)
+> data into insightful graphs and visualizations.
+>
+> {{% caption %}}[Grafana documentation](https://grafana.com/docs/grafana/latest/introduction/){{% /caption %}}
+
+For the most performant queries, use SQL and the
+[Flight SQL protocol](https://arrow.apache.org/blog/2022/02/16/introducing-arrow-flight-sql/)
+to query InfluxDB.
+
+
+## Build the Grafana FlightSQL plugin
```

Comment:
We're going to swap these instructions out with some instructions downloading and installing a release instead. Should greatly simplify the setup.

I'll supply a sample Dockerfile, too.

Reply:
Opened a PR against this branch here: https://github.com/influxdata/docs-v2/pull/4724